server/ws/server.js

```diff
@@ -0,0 +1,22 @@
+import { WebSocketServer } from 'ws';
+import { Worker } from 'worker_threads';
+
+var server = new WebSocketServer({
+  port: 8080,
+});
+
+const start = async () => {
+  let nextSessionId = 0;
+  server.on('connection', async (ws) => {
```

Comment:
If the connection drops is there any way to pick back up?

Reply:
Not at the moment -- I used the connection drop to reset the session.

But it could be done with a TTL.